_posts/2016/2016-05-25-spring-io.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Spring I/O 2016"
+date: "2016-05-25 09:00:00"
+image: blog-header/springio.jpg
```

Comment:
Please make sure that the aspect ratio of the picture is 2:1 and the width is 850px.